### PR TITLE
libgweather 3.16.1 (new formula)

### PR DESCRIPTION
**Pull request by tschoonj**

### CODE REVIEW

File: Library/Formula/libgweather.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                          "--disable-schemas-compile"
+    system "make", "install"
+  end
+
+  def post_install
+    system "#{Formula["glib"].opt_bin}/glib-compile-schemas", "#{HOMEBREW_PREFIX}/share/glib-2.0/schemas"
+  end
+
+  test do
+    system "false"
```
Comment (bfontaine) [inline Library/Formula/libgweather.rb]: Please add a test :wink:

**Comment by tschoonj** [comment]
Apologies! Was too fast this time :-)

**Comment by bfontaine** [comment]
@tschoonj No worries :)

File: Library/Formula/libgweather.rb [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  depends_on "intltool" => :build
+  depends_on "gtk+3"
+  depends_on "geocode-glib"
+  depends_on "libsoup"
+  depends_on "gobject-introspection"
+  depends_on "vala" => :optional
+
+  def install
+    # ensures that the gobject-introspection files remain within the keg
+    inreplace "libgweather/Makefile.in", "@HAVE_INTROSPECTION_TRUE@girdir = $(INTROSPECTION_GIRDIR)",
```
Comment (DomT4) [inline Library/Formula/libgweather.rb]: Just a nit but when `inreplace`-ing the same file, you can just do:

```ruby
    inreplace "libgweather/Makefile.in" do |s|
      s.gsub! "@HAVE_INTROSPECTION_TRUE@girdir = $(INTROSPECTION_GIRDIR)",
              "@HAVE_INTROSPECTION_TRUE@girdir = $(datadir)/gir-1.0"
      s.gsub! "@HAVE_INTROSPECTION_TRUE@typelibdir = $(INTROSPECTION_TYPELIBDIR)",
              "@HAVE_INTROSPECTION_TRUE@typelibdir = $(libdir)/girepository-1.0"
    end
```

I'll fix it on pull here.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/39c8f78b95aa861d6ada1e6132b029eeca842056. Thanks again Tom!

**Pull request closed without merging**